subscriptions/tests.py

```diff
@@ -774,6 +775,144 @@ def test_send_message_task_to_mother_text(self):
 
         post_save.disconnect(fire_sub_action_if_new, sender=Subscription)
 
+    @override_settings()
```

Comment:
Not sure if this is necessary?

Reply:
If you look at line 800, you'll see that we're overriding a settings variable. I can't do it at the top here, because I don't know what the messageset ID is going to be in advance.

I *think* this handles restoring the original settings for me, so that the setting isn't permanently changed for all tests.

Reply:
Ah cool ok then.